flowup-domcinTask/src/app/models/account.ts

```diff
@@ -0,0 +1,6 @@
+export class Account {
```

Comment:
```suggestion
export interface Account {
```

An interface is better because it actually doesn't produce anything in the transplied Javascript, it's just used by Typescript to check object fields. A class is something you would have to instantiate with `new` and implement a constructor.